lib/types/entry.ts

```diff
@@ -62,6 +65,7 @@ export interface LocalizedEntry<Fields extends FieldsType, Locale extends Locale
       [LocaleName in Locale]?: Fields[FieldName]
     }
   }
+  metadata: Metadata
```

Comment:
Something we have to test. I think `metadata` is not available for `Link`

Reply:
It just looks like this is in `Link` due to GH diffing view. But it's actually on `LocalizedEntry` :-)